tensorflow/core/common_runtime/eager/kernel_and_device.cc

```diff
@@ -360,8 +408,8 @@ Status KernelAndDeviceFunc::Run(
   {
     profiler::TraceMe activity(
         [&] {
-          return absl::StrCat("FunctionRun#name=", name(),
-                              ",id=", opts->step_id, "#");
+          return absl::StrCat("FunctionRun#name=", name(), ",id=",
```

Comment:
what's going on with the formatting changes? Can you avoid changes to unrelated code?

Reply:
Oh my bad, for some reason, clang-format did this change. I'll revert

Reply:
fixed in c2b40a2f7c1d3f11b48ced0efe7cd5949f1faed3